sourmash/lca/lca_db.py

```diff
@@ -27,7 +29,7 @@ def get(self):
     return property(get)
 
 
-class LCA_Database(Index):
+class LCA_Database(RustObject):
```

Comment:
Let's keep both `RustObject` and `Index` here (`RustObject` is required/convenient for Rust FFI, `Index` is an [abstract base class][0] with methods that every `Index` must support.

[0]: https://docs.python.org/3/glossary.html#term-abstract-base-class